src/content/architecture/globalnet/_index.en.md

```diff
@@ -0,0 +1,77 @@
+
+Once configured, each service and pod that requires cross-cluster access is allocated an IP, `GlobalIp` from this `GlobalCIDR` and it is annotated on the Pod/Service object. This `GlobalIp` is used for all cross-cluster communication to and from to this Pod/Service. Routing and IPTable/OVS/OVN rules are configured to use this IP for ingress and egress. All address translations occur on the Gateway node.
+
+![Figure 1 - Proposed solution](/images/globalnet/overlappingcidr-solution.png)
+
+### submariner-globalnet
+
+Submariner GlobalNet is a component that provides cross-cluster connectivity between pods and services using their GlobalIps. Compiled as binary `submariner-globalnet`, it is the controller that is responsible for maintaining a pool of global IPs, allocating IPs from the globalIp pool to pods and services, annotating services and pods with their global Ip, and configuring the required rules on gateway node to provide cross-cluster connectivity using GlobalIps. It mainly consists of two key components: the IP Address Manager and Globalnet.
+
+#### IP Address Manager (IPAM)
+
+The IP Address Manager (IPAM) component does the following:
+
+* Creates a pool of IP addresses based on `GlobalCIDR` configured on cluster.
+* On creation of a Pod/Service, allocates a GlobalIp from the GlobalIp pool.
+* Annotates the Pod/Service with `submariner.io/GlobalIp=<global-ip>`.
+* On deletion of a Pod/Service, releases its GlobalIp back to the pool.
+
+#### Globalnet
+
+This component is responsible for programming the routing entries and IPTable rules and does the following:
+
+* Creates initial IPTables chains for Globalnet rules.
+* Whenever a pod is annotated with Global IP, creates an egress SNAT rule to convert source ip from PodIp to pod's GlobalIp on the Gateway Node.
+* Whenever a service is annotated with Global IP, creates an ingress rule to direct all traffic destined to service's GlobalIp to the service's `kube-proxy` IPTables chain which in turn directs traffic to service's backend pods.
+* On deletion of pod/service, clean up the rules from the gateway node.
+
+Globalnet currently relies on `kube-proxy` and thus will only work with deployments that use `kube-proxy`.
+
+### Service Discovery - Lighthouse
+
+Connectivity is only part of the solution as pods still need to know the IPs of services on remote clusters.
+
+This is achieved by enhancing [lighthouse](https://github.com/submariner-io/lighthouse) with support for GlobalNet. The Lighthouse controller adds the service's GlobalIp to the `MultiClusterService` object that is distributed to all clusters. The [lighthouse plugin](https://github.com/submariner-io/lighthouse/tree/master/plugin/lighthouse) then uses the service's GlobalIp when replying to DNS queries for the service.
+
+## Installation
+
+TBD.
+
+## Testing
+
+Refer [testing guide](testing.md) for instructions on how to test, same steps apply.
```

Comment:
This doesn't apply here.

Reply:
Actually this is taken as is from the globalnet readme we had in Submariner repo. I can remove it, no issues. Thanks for reviewing @mangelajo